t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before

```diff
@@ -0,0 +1,2 @@
+rsync --timeout=3600 -tlp4 --specials obspublish::openqa/openSUSE:Factory:ToTest/appliances/x86_64/*openSUSE-Tumbleweed-20210701-WSL.x86_64-21153.2.2.0-Snapshot20210701.appx /var/lib/openqa/factory/other/openSUSE-Tumbleweed-20210701-WSL.x86_64-21153.2.2.0-Snapshot20210701.appx
```

Comment:
kiwi-images-wsl is still not in the path, I will have a look how to fix it

Reply:
I cannot push to your repo, so could you add this commit and all should be good 
https://github.com/andrii-suse/openqa-trigger-from-obs/commit/acd7da493

Reply:
Done